edgelet/iotedge/src/client.rs

```diff
@@ -146,10 +160,7 @@ impl ModuleRuntime for MgmtClient {
     async fn get(&self, _id: &str) -> anyhow::Result<(Self::Module, ModuleRuntimeState)> {
         unimplemented!()
     }
-    async fn start(&self, _id: &str) -> anyhow::Result<()> {
-        unimplemented!()
-    }
-    async fn stop(&self, _id: &str, _wait_before_kill: Option<Duration>) -> anyhow::Result<()> {
```

Comment:
So, we are saying start/stop are now unimplemented?

Reply:
No, they are now implemented. Restart is the one that is now unimplemented.

Reply:
or is this just internal?